src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -34,7 +34,7 @@ public static Client[] getSampleClients() {
                 LocalDate.parse("2023-11-11", DateTimeUtil.DEFAULT_DATE_PARSER));
         Client[] people = new Client[]{
             new Client(new Name("Alex Yeoh"),
-                        new Company("Tech Innovations Pte Ltd"),
```

Comment:
@reginateh Just to double check is this data file bundled with the jar file? If it is I think we're not allowed to change it 
![image](https://github.com/user-attachments/assets/f6dd212f-688c-401a-81d3-e5b5a4dec707)